app/backend/src/couchers/servicers/events.py

```diff
@@ -734,6 +736,21 @@ def ListMyEvents(self, request, context):
                     ),
                 )
                 where_.append(EventOccurrenceAttendee.user_id != None)
+            if include_my_communities:
+                my_communities = (
+                    session.execute(
+                        select(Node.id)
+                        .join(Cluster, Cluster.parent_node_id == Node.id)
+                        .join(ClusterSubscription, ClusterSubscription.cluster_id == Cluster.id)
+                        .where(ClusterSubscription.user_id == context.user_id)
+                        .where(Cluster.is_official_cluster)
+                        .order_by(Node.id)
+                        .limit(100000)
```

Comment:
I kinda copied this from `threads.py` where 100000 is an arbitrarily "large enough" (?) number so it should be sufficient to include all communities the user is in

Reply:
Whenever we convert this into a subquery, the list of clusters will not be extracted from the database and this arbitrary limit will go away. So it is fine for now :)